modules/mapper-extras/src/test/java/org/elasticsearch/index/mapper/RankFeatureMetaFieldMapperTests.java

```diff
@@ -55,4 +57,14 @@ public void testBasics() throws Exception {
         assertEquals(mapping, mapper.mappingSource().toString());
         assertNotNull(mapper.metadataMapper(RankFeatureMetaFieldMapper.class));
     }
+
+    public void testDocumentParsingFailsOnMetaField() throws Exception {
```

Comment:
Great way to test!
nit: could you add a very brief comment explaining why this proves the metadata field from the plugin was correctly registered? Makes it easier to later track why tests are there and what they check.

Reply:
@cbuescher Thank you for the review.
The last comment addressed in ac6a4bc